release-note-awso (#2189)
SumoLogic/sumologic-documentation: blog-service/2023-04-25-awso.md

---
title: April 25, 2023 (Observability)
image: https://www.sumologic.com/img/logo.svg
keywords:
  - apps
  - app catalog
  - saas cloud
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

> Update - We're excited to announce the release of the AWS Observability 2.6. Here are some of the features the new version offers:

Reviewer: ```suggestion
We're excited to announce the release of AWS Observability 2.6. Here are some of the features the new version offers:
```

- Support for Amazon SQS - we've added out-of-the-box dashboards with important information about queue and message statistics, including 4 new dashboards and predefined monitors.

> - Updates to AWS Lambda dashboards to include Lambda Telemetry API metrics.

Reviewer: ```suggestion
* **AWS Lambda dashboards**. Updated to include Lambda Telemetry API metrics for improved observability.
```

> * **Entity Inspector KPIs*. New KPIs were added to help users gain better visibility into their entities.

Reviewer: ```suggestion
* **Entity Inspector KPIs**. New KPIs were added to help users gain better visibility into their entities.
```

> - Added evaluation delay to the out-of-the-box monitors to improve the accuracy and deviation detection.

Reviewer: ```suggestion
* **Out-of-the-box monitors**. Added evaluation delay for improved accuracy and deviation detection.
```

> - The Node.js runtime environment for the AWS Observability Lambda functions has been updated to the latest available versions.

Reviewer: ```suggestion
* **AWS Observability Lambda functions**. Updated to use the latest available Node.js runtime environment.
```
